wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_discrete.py

```diff
@@ -0,0 +1,101 @@
+__all__ = ["YaqdWrightFilterWheelsDiscrete"]
+
+import asyncio
+from typing import Dict, Any, List
+from yaqd_core import DiscreteHardware, aserial
+
+from .__version__ import __branch__
+
+class YaqdWrightFilterWheelsDiscrete(DiscreteHardware):
+    _kind = "wright-filter-wheels-discrete"
+    _version = "0.1.0" + f"+{__branch__}" if __branch__ else ""
+    traits: List[str] = ["uses-uart","uses-serial","is-homeable"]
+    defaults: Dict[str, Any] = {"baud_rate": 57600}
+    
+    def __init__(self, name, config, config_filepath):
+        super().__init__(name, config, config_filepath)
+        self._motornum=config["motor"]
+        self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
+        self.microstep=1
+        self._steps_per_rotation=400
+        self._current_identifier=""
+        self._offset=0  # try incorporating any adjustments to TOML before adjusting this
+        self._identifiers=config["identifiers"]
+        self.home()
+
+
+    def _load_state(self, state):
+        """Load an initial state from a dictionary (typically read from the state.toml file).
+
+        Must be tolerant of missing fields, including entirely empty initial states.
+
+        Parameters
+        ----------
+        state: dict
+            The saved state to load.
+        """
+        super()._load_state(state)
+        # This is an example to show the symetry between load and get
+        # If no persistent state is needed, these unctions can be deleted
+        self.value = state.get("value", 0)
+
+    def get_state(self):
+        state = super().get_state()
+        state["value"] = self.value
+        return state
+
+    def set_offset(self,value):
+        self._offset=value
```

Comment:
Why is offset needed?

If it is kept, it should be in one of config or state, as it should survive restart

If it is in config, there should not be a `set_offset` function

Reply:
I can see offset being more use in the continuous case than the discrete case, as continuous will have a zero mark which may not align with the homing flag

Reply:
This could be a way to crudely tweak the wheel.   It is difficult knowing just how precise the positions have to be for users using this wheel.  Offset could be saved in state TOML.

Reply:
Why do you need to "crudely tweak the wheel", i still do not have a good understanding of why this is here.

Reply:
offset is removed but there is an intent to leave a placeholder or two as  reminder that it could be placed back in if needed.    `_approximate_position` would be a simple means of implementation, thought it has been removed